napari/layers/_vectors_layer/view/properties.py

```diff
@@ -0,0 +1,85 @@
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QLabel, QComboBox, QDoubleSpinBox, QSpinBox, QGridLayout
+import numpy as np
+import scipy.signal as signal
+
+from ..._base_layer import QtLayer
+
+
+class QtVectorsLayer(QtLayer):
+
+    def __init__(self, layer):
+        super().__init__(layer)
+
+        self.layer.events.emit_avg.connect(self.change_avg)
+        self.layer.events.emit_len.connect(self.change_len)
+
+        # vector color adjustment and widget
+        face_comboBox = QComboBox()
+        colors = self.layer._colors
+        for c in colors:
+            face_comboBox.addItem(c)
+        index = face_comboBox.findText(self.layer.color, Qt.MatchFixedString)
+        if index >= 0:
```

Comment:
what's this for? when is the index <=0 ?

Reply:
This happens if the color is not found in the list of supported colors, and is something that should probably be explained in the markers and shapes layer too